### DISCUSSION TITLE
Getting error message in log - "Could not connect to web hook service"

### DISCUSSION THREAD

**Discussion by seancpark** [iklein99/homebridge-smartthings#251] [Q&A]
Any insight will be greatly appreciated, have been having an issue with Nest Learning thermostats going inactive in HK so I took a look at how things are running on the HOOBS and when looking at the log I noticed continual "could not connect" error messages.

- SmartThings Ik BridgeSmartthings Plug (IK)ERRORCould not connect to web hook service: AxiosError: Request failed with status code 401.  Will retry

FYI this has been running fine since August '23.

Thoughts? Thanks!

**Comment by iklein99** [comment]
This indicates that you have the webhooktoken config set, but your token is unrecognized.  Did you sign up for the webhook service?  If so, send me the user name or email address you used when you did.

**Comment by seancpark** [comment]
Yes, have the service - seancpark@gmail.com

**Comment by iklein99** [comment]
Sean, I sent you, privately, your webhook access token.  However, I can see in the logs your plugin is successfully accessing the service.  That said, I see no events coming from smartthings.

I'm not sure why you would see 401 errors as I can see you are successfully accessing the service.

**Comment by seancpark** [comment]
So the 401 errors have disappeared, but now it's shifted to "could not connect" with timeout error.
![Screenshot 2024-03-05 at 5 17 53 AM](https://github.com/iklein99/homebridge-smartthings/assets/89873123/aedf42db-1448-4e90-b2e5-a0d7ff1fa0dd)

**Comment by iklein99** [comment]
Sean,

I’m seeing your homebridge requests on the server and I’m seeing events for you, so all is working ok.  Not sure why you are seeing those messages, but there is nothing going on with the service or your plugin.  Those errors indicate to me that you may be having network issues on your side of things or with your network provider.  I am seeing none of those errors when I check my client and I see normal operation on the server.

**Reply by seancpark** [reply]
Ok, thanks for taking a look, we have 1G symmetrical fiber and I checked the log and it seems to be all good. I'll keep an eye on it and see if it happens again, but it's been a while so I'll keep my fingers crossed. 

Again, thanks for the assistance!
